### PR TITLE
Added workspace owner contact info to welcome in workspace email

### PR DESCRIPTION
**Pull request by martin-morek**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4953,6 +4953,12 @@
       "resolved": "https://registry.npmjs.org/@types/unist/-/unist-2.0.3.tgz",
       "integrity": "sha512-FvUupuM3rlRsRtCN+fDudtmytGO6iHJuuRKS1Ss0pG5z8oX0diNEw94UEL7hgDbpN94rgaK5R7sWm6RrSkZuAQ=="
     },
+    "@types/url-parse": {
```
Comment (afoerster) [inline package-lock.json:4956]: This change shouldn't be included. We should upgrade packages soon though

**Review by afoerster — COMMENTED** [review]
@martin-morek can you show a screenshot of what this looks like when it's been run?

**Comment by martin-morek** [comment]
New email look: 

Screenshot 2020-05-26 at 09 52 37

File: common/src/main/scala/io/phdata/services/EmailService.scala [hunk 2]
```diff
@@ -31,12 +31,15 @@ class EmailServiceImpl[F[_]: Effect](context: AppContext[F], workspaceService: W
...
+      owner <- context.lookupLDAPClient.findUserByDN(workspace.requestedBy)
       values = Map(
         "roleName" -> memberRoleRequest.role.get.show,
         "resourceType" -> memberRoleRequest.resource,
         "workspaceName" -> workspace.name,
         "uiUrl" -> resolveUiUrl,
-        "workspaceId" -> workspaceId
+        "workspaceId" -> workspaceId,
+        "ownerName" -> owner.name,
+        "ownerEmail" -> owner.email.getOrElse("Email not provided")
```
Comment (afoerster) [inline common/src/main/scala/io/phdata/services/EmailService.scala]: Print out error message if there was one, or print out error saying the owner email was not found

**Review by afoerster — APPROVED** [review]
LGTM

**Pull request merged**